lib/packages/docutils/rst.nim

```diff
@@ -754,14 +758,14 @@ proc rstnodeToRefname(n: PRstNode): string =
   var b = false
   rstnodeToRefnameAux(n, result, b)
 
-proc findSub(p: var RstParser, n: PRstNode): int =
+proc findSub(s: RstSharedState, n: PRstNode): int =
   var key = addNodes(n)
   # the spec says: if no exact match, try one without case distinction:
-  for i in countup(0, high(p.s.subs)):
-    if key == p.s.subs[i].key:
+  for i in countup(0, high(s.subs)):
```

Comment:
it's ok to cleanup lines that you are editing; `0..high(s.subs)` is shorter and better

Reply:
agree but let us to do it in follow-up